### DISCUSSION TITLE
"PowerToys" is ambiguous?

### DISCUSSION THREAD

**Discussion by bsitpro** [microsoft/winget-cli#1573] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/92312703/136792275-93a357d9-efbf-4bc3-a37a-5b75779e5916.png)

I love winget but this sort of thing makes life difficult.  Can someone help me understand why this happens and how to install powertoys?

**Comment by Masamune3210** [comment, marked as answer]
Known issue with msstore source, fix is being worked on behind the scenes, either disable the msstore source for now or use --id to refer to a specific id when installing

**Reply by denelon** [reply]
Here are some [customization tips](https://devblogs.microsoft.com/commandline/windows-package-manager-1-1/#customization-tips).
